### DISCUSSION TITLE
콜백 사용 샘플이 있을까요?

### DISCUSSION THREAD

**Discussion by jgw1413** [naverz/zepeto-studio-kor#1131] [5. Q&A]
멀티 월드를 제작하면서 콜백이 필요하다고 느껴져서 그러는데 샘플이나 예제가 있을까요?

ZEPETO Studio에 안보여서 질문드립니다 ㅠ,ㅠ

**Comment by ZdevMonZ** [comment]
결국 타입스크립트라서 타입스크립트 콜백쪽 자료 보심 도움이 될것 같네요 
https://woojin.tistory.com/m/22

**Reply by jgw1413** [reply]
제페토 월드만들면서 처음보는 스크립트라 많이 걱정했는데 생각보다 자료가 많았군요! 감사합니다! ^^
